pkg/products/monitoringcommon/blackbox.go

```diff
@@ -0,0 +1,66 @@
+package monitoringcommon
+
+import (
+	"context"
+	"fmt"
+	v1alpha12 "github.com/integr8ly/integreatly-operator/apis/v1alpha1"
```

Comment:
nit: could this just be v1alpha1

Reply:
Turns out the function in this file is not used anywhere. File can be removed.